src/ProjectTemplates/GeneratedContent.targets

```diff
@@ -44,9 +44,9 @@
       <TemplatePackageVersion_MicrosoftExtensionsAI>9.9.1</TemplatePackageVersion_MicrosoftExtensionsAI>
       <TemplatePackageVersion_MicrosoftExtensionsAIOpenAI>9.9.1-preview.1.25474.6</TemplatePackageVersion_MicrosoftExtensionsAIOpenAI>
 
-      <TemplatePackageVersion_Aspire>9.4.0</TemplatePackageVersion_Aspire>
-      <TemplatePackageVersion_Aspire_Preview>9.4.0-preview.1.25378.8</TemplatePackageVersion_Aspire_Preview>
-      <TemplatePackageVersion_AzureAIOpenAI>2.3.0-beta.1</TemplatePackageVersion_AzureAIOpenAI>
+      <TemplatePackageVersion_Aspire>9.5.0</TemplatePackageVersion_Aspire>
+      <TemplatePackageVersion_Aspire_Preview>9.5.0-preview.1.25474.7</TemplatePackageVersion_Aspire_Preview>
+      <TemplatePackageVersion_AzureAIOpenAI>2.3.0-beta.2</TemplatePackageVersion_AzureAIOpenAI>
```

Comment:
Not changed in your PR but do you also care about updating the communitytoolkit aspire version? There is a newer version of that as well.